dimitriosv/week4/app/stores/list-store.js

```diff
@@ -0,0 +1,47 @@
+var Dispatcher =  require('../dispatcher/dispatcher.js');
+var Constants = require('../constants');
+var BaseStore = require('./base');
+
+var _list = [{listId: 1, listName: "first list1"},{listId: 2, listName: "second list2"}];
+var _totalElements = 2;
+
+
+function deepCopy(itemToCopy) {
+    var listCopy = JSON.parse(JSON.stringify(itemToCopy));
+    return listCopy;
+}
+
+// reads
+var store = Object.assign({}, BaseStore, {
+	getAll: function() {
+		return deepCopy(_list);
+	},
+    getTotalElements: function() {
```

Comment:
This is an anti-pattern to me. No need for this.
You can count the elements from getAll()